apps/prairielearn/elements/pl-symbolic-input/pl-symbolic-input.py

```
@@ -499,81 +527,112 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
+            )
+            result = _apply_parse_constraints_to_sympy_expr(
+                sympy_expr,
+                variables=variables,
+                custom_functions=custom_functions,
+                assumptions=assumptions_dict,
+                allow_complex=allow_complex,
+                allow_sets=allow_sets,
+                allow_trig=allow_trig,
+                simplify_expression=simplify_expression,
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)